src/components/sections/ShopLinks/ShopLink/ShopLink.tsx

```diff
@@ -0,0 +1,66 @@
+// libs imports
+import React from 'react';
+import { Link } from 'react-router-dom';
+// local imports
+import css from './ShopLink.module.scss';
+
+//! chagne later
+// type imagesProps = {
+//   mabile: string;
+//   tablet: string;
+//   // desktop: string;
+// };
+
+// type ShopLinkProps = {
+//   name: string;
+//   images: any;
+//   target: string;
+// };
+
+type Device = {
```

Comment:
extract types into separate structure 
/types
 - index.ts
 - common.ts
 - shopLinks.ts 
 
 Or somewhere locally to the component into a separate file.
 
 imho
 1 interface inside a react component file is good, but two or more is a little overwhelming.

Reply:
fixed on another branch - https://github.com/myroslavsuprun/audiophile/tree/feature/form-elements